psalm.xml.dist

```diff
@@ -17,12 +19,6 @@
         </ignoreFiles>
     </projectFiles>
 
-    <stubs>
-        <file name="stubs/BSON/Document.stub.php"/>
-        <file name="stubs/BSON/Iterator.stub.php"/>
-        <file name="stubs/BSON/PackedArray.stub.php"/>
```

Comment:
Does Psalm 6 now infer type info for these classes, or are these obsolete because the stubs are now bundled in Psalm directly? I perused the [release notes](https://github.com/vimeo/psalm/releases) from 6.0+ but didn't see anything of note.

Reply:
I think they are all part of the stub of a long time: https://github.com/vimeo/psalm/pull/10016